### PR TITLE
Changed NV software trigger

### PR DESCRIPTION
**Pull request by WenzDaniel**
## What does the code in this PR do / what does it improve?
This PR prepares the NV software trigger for the deletion of `raw_records_nv` after processing on the EBs. 

If we want to merge this PR and apply the trigger we must make a new straxen version 3.0.0!

### CODE REVIEW

**Comment by yuema137** [comment]
@WenzDaniel, I converted it to a draft as there are still some unsolved errors in the auto checks. Please let me know when this PR is ready. Thanks!

**Comment by WenzDaniel** [comment]
Can be merged after https://github.com/XENONnT/straxen/pull/1228
Once both are merged I would like to do some last test processing checks.

**Review by yuema137 — APPROVED** [review]

**Pull request merged**
